redis/client.py

```diff
@@ -358,6 +362,15 @@ def set_response_callback(self, command: str, callback: Callable) -> None:
         """Set a custom Response Callback"""
         self.response_callbacks[command] = callback
 
+    def _cache_invalidation_process(
+        self, data: List[Union[str, Optional[List[str]]]]
+    ) -> None:
+        if data[1] is not None:
```

Comment:
What happens *if* an IndexError happens?

Reply:
it shouldn't happen, if it happens it's a parsing error and need to be raised